src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -31,10 +34,14 @@ class Files(SdkApi):
     connection
         connection object
     """
+    dsn: Datasets
```

Comment:
I'm wondering if we want to rename this variable to `ds` to be consistent with the API endpoint? (which is  `/zosmf/restfiles/ds` for data sets)

Reply:
I think the suggestion to change `dsn` with `ds` across the board makes sense for consistency.